utils/check_config_docstrings.py

```diff
@@ -51,10 +51,12 @@ def get_checkpoint_from_config_class(config_class):
     config_source = inspect.getsource(config_class)
     checkpoints = _re_checkpoint.findall(config_source)
 
-    for checkpoint in checkpoints:
-        # Each `checkpoint` is a tuple of a checkpoint name and a checkpoint link.
-        # For example, `('bert-base-uncased', 'https://huggingface.co/bert-base-uncased')`
-        ckpt_name, ckpt_link = checkpoint
+    # Each `checkpoint` is a tuple of a checkpoint name and a checkpoint link.
+    # For example, `('bert-base-uncased', 'https://huggingface.co/bert-base-uncased')`
+    for ckpt_name, ckpt_link in checkpoints:
+        # allow the link to end with `/`
+        if ckpt_link.endswith("/"):
+            ckpt_link = ckpt_link[:-1]
```

Comment:
To allow link with ending `/` work, like
```python
https://huggingface.co/BridgeTower/bridgetower-base/
```

Reply:
Also, previously, `checkpoint` is used for looping, which is not good: a few lines below this place, we have

```python
        if ckpt_link == ckpt_link_from_name:
            checkpoint = ckpt_name
            break
```
We only want to return `checkpoint` under this verified condition, otherwise, it should return `None`.